packages/apm/src/integrations/tracing.ts

```diff
@@ -355,6 +370,18 @@ export class Tracing implements Integration {
       };
     }
 
+    logger.log(`[Tracing] pushActivity: ${name}#${Tracing._currentIndex}`);
+    logger.log('[Tracing] activies count', Object.keys(Tracing._activities).length);
+    if (options && options.autoPopAfter) {
+      logger.log(`[Tracing] auto pop of: ${name}#${Tracing._currentIndex} in ${options.autoPopAfter}ms`);
+      const index = Tracing._currentIndex;
+      setTimeout(() => {
+        Tracing.popActivity(index, {
```

Comment:
Is popping same activity twice safe?

Reply:
Yes, it doesn't do anything if it no longer exists.